mabl snippets/scrollToElement.js

```diff
@@ -6,12 +6,23 @@
  */
 function mablJavaScriptStep(mablInputs, callback) {
 
+let result = '';
  
-  // grab the element
-  let element = document.querySelector('CSS selector of element');
-  
-  // True scrolls to the top of the element, false scrolls to the bottom
-  element.scrollIntoView(false);
+// get element location by passing it a Journey parameter named rowID
+let element = document.querySelector('CSS selector' + mablInputs.variables.user.rowID);
 
-  callback('Done');
+if (element !== null) {
+ 
+// false will scroll to the bottom of the element whereas true will scroll to the top
+element.scrollIntoView(false);
```

Comment:
Can you add some indentation here and other places? You can select the text in VS Code, right click and select to format it.

Reply:
check out the new version